projects/packages/videopress/src/client/block-editor/blocks/video/components/playback-panel/index.tsx

```diff
@@ -45,27 +34,34 @@ export default function PlaybackPanel( { attributes, setAttributes }: VideoContr
 	return (
 		<PanelBody title={ __( 'Playback', 'jetpack-videopress-pkg' ) }>
 			<ToggleControl
-				label={ renderControlLabelWithTooltip(
-					__( 'Autoplay', 'jetpack-videopress-pkg' ),
-					/* translators: Tooltip describing the "autoplay" option for the VideoPress player */
-					__( 'Start playing the video as soon as the page loads', 'jetpack-videopress-pkg' )
-				) }
+				label={ __( 'Autoplay', 'jetpack-videopress-pkg' ) }
 				onChange={ handleAttributeChange( 'autoplay' ) }
 				checked={ autoplay }
-				help={ __(
-					'Note: Autoplaying videos may cause usability issues for some visitors.',
-					'jetpack-videopress-pkg'
-				) }
+				help={
+					<>
+						<p>
+							{ __(
+								'Start playing the video as soon as the page loads.',
+								'jetpack-videopress-pkg'
+							) }
+						</p>
+						{ autoplay && (
+							<p>
+								{ __(
+									'Note: Autoplaying videos may cause usability issues for some visitors.',
+									'jetpack-videopress-pkg'
+								) }
+							</p>
+						) }
+					</>
+				}
 			/>
 
 			<ToggleControl
-				label={ renderControlLabelWithTooltip(
-					__( 'Loop', 'jetpack-videopress-pkg' ),
-					/* translators: Tooltip describing the "loop" option for the VideoPress player */
-					__( 'Restarts the video when it reaches the end', 'jetpack-videopress-pkg' )
-				) }
+				label={ __( 'Loop', 'jetpack-videopress-pkg' ) }
 				onChange={ handleAttributeChange( 'loop' ) }
 				checked={ loop }
+				help={ __( 'Restarts the video when it reaches the end', 'jetpack-videopress-pkg' ) }
```

Comment:
To keep consistency, can we add a period `.` at the end of the labels? I saw some with and some without, but checking on other blocks and elementos from the UI, using the full stop seems to be a common practice.

```suggestion
				help={ __( 'Restarts the video when it reaches the end.', 'jetpack-videopress-pkg' ) }
```